### PR TITLE
Add Confirmation Dialog to Portal Submission

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

File: portal-frontend/src/app/features/edit-submission/review-and-submit/submit-confirmation-dialog/submit-confirmation-dialog.component.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<div mat-dialog-title>
+  <h4>Submit Application</h4>
+</div>
+
+<div mat-dialog-content class="content">
+  <p>
+    Your application will be submitted to the [L/FNG]. After submission, you will be auto-directed to complete the local
```
Comment (sandratoh) [inline portal-frontend/src/app/features/edit-submission/review-and-submit/submit-confirmation-dialog/submit-confirmation-dialog.component.html:7]: I remember the [L/FNG] was just a design placeholder, and that it is supposed to be the name of the actual government. I would recommend confirming this with the design team

File: portal-frontend/src/app/features/edit-submission/review-and-submit/submit-confirmation-dialog/submit-confirmation-dialog.component.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  </p>
+  <div>
+    <div class="step-controls">
+      <button mat-stroked-button color="primary" [mat-dialog-close]="false">cancel</button>
+      <button
+        mat-flat-button
+        color="primary"
+        [disabled]="!checkbox1.checked || !checkbox2.checked || !checkbox3.checked"
+        [mat-dialog-close]="true"
+      >
```
Comment (sandratoh) [inline portal-frontend/src/app/features/edit-submission/review-and-submit/submit-confirmation-dialog/submit-confirmation-dialog.component.html:38]: is there supposed to be `(click)=""` somewhere to call `onSubmit` and `onCancel`?

Comment (dhaselhan) [reply portal-frontend/src/app/features/edit-submission/review-and-submit/submit-confirmation-dialog/submit-confirmation-dialog.component.html:38]: Nope! I have deleted onSubmit and onCancel, [mat-dialog-close]="true" sets the button to close the dialog and passes whatever value in the brackets to any subscribers.

File: portal-frontend/src/app/features/edit-submission/review-and-submit/submit-confirmation-dialog/submit-confirmation-dialog.component.html (hunk 1 above)
Comment (dhaselhan) [reply portal-frontend/src/app/features/edit-submission/review-and-submit/submit-confirmation-dialog/submit-confirmation-dialog.component.html:7]: Fixed!

**Review by sandratoh — APPROVED** [review]
LGTM 👍

**Pull request merged**